assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -58,17 +63,44 @@ const subHeadingText = {
     monthly or one-time contribution today`,
 };
 
-function contribAttrs(isoCountry: IsoCountry): ContribAttrs {
+const contribCtaText = {
+  RECURRING: 'Contribute with card or PayPal',
+  ONE_OFF: 'Contribute with debit/credit card',
+};
+
+function contribAttrs(isoCountry: IsoCountry, contribType: Contrib): ContribAttrs {
   return {
     heading: 'contribute',
     subheading: subHeadingText[isoCountry],
-    ctaText: 'Contribute',
+    ctaText: contribCtaText[contribType],
     modifierClass: 'contributions',
     ctaLink: '',
     showPaymentLogos: false,
   };
 }
 
+function showPayPal(props: PropTypes) {
+  switch (props.contribType) {
```

Comment:
Is there a special reason that you use a switch here? I looks like you can use a simple if.

Reply:
Personal preference mainly. The `switch` structure always looks cleaner to me than `if (thing === thing) {...} else {...}`. Admittedly in this case it could be either, it tends to help more when there are additional matches to catch.